src/types.ts

```diff
@@ -147,6 +148,30 @@ export interface HyperCube extends Omit<EngineAPI.IHyperCube, 'qDimensionInfo' |
   qMeasureInfo: ExtendedNxMeasureInfo[];
 }
 
+interface InlineDimensionDef extends EngineAPI.INxInlineDimensionDef {
+  textAlign: TextAlign;
+}
+interface InlineMeasureDef extends EngineAPI.INxInlineMeasureDef {
+  textAlign: TextAlign;
+}
+interface AttributeExpressionProperty extends EngineAPI.INxAttrExprDef {
```

Comment:
same here, it is a collection of properties

Reply:
Fixed